### PR TITLE
Update to ember 3.16

### PR DESCRIPTION
**Pull request by svsalvador**

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -1,51 +1,60 @@
...
+  },
+  rules: {
+    'ember/no-jquery': 'error'
+  },
+  overrides: [
+    // node files
+    {
+      files: [
+				'.huskyrc.js',
+				'.commitlintrc.js',
```
Comment (adrigzr) [inline .eslintrc.js]: Do a `npm run lint:js -- --fix` to fix this.

File: package.json [hunk 2]
```diff
@@ -18,66 +18,73 @@
...
   "scripts": {
-    "build": "ember build",
-    "lint:hbs": "ember-template-lint .",
-    "lint:js": "eslint --max-warnings 0 ./*.js addon app config tests",
+    "build": "ember build --environment=production",
     "start": "ember serve",
     "test": "COVERAGE=true ember test",
-    "test:all": "ember try:each",
-    "check-coverage": "istanbul check-coverage coverage/coverage-final.json --statements 100 --functions 100 --branches 95 --lines 100",
+    "check-coverage": "istanbul check-coverage coverage/coverage-final.json --statements 96 --functions 90 --branches 90 --lines 96",
```
Comment (adrigzr) [inline package.json]: 100? :)

File: .travis.yml [hunk 3]
```diff
@@ -45,24 +54,15 @@ jobs:
...
-      env: EMBER_TRY_SCENARIO=ember-lts-2.18
-    - env: EMBER_TRY_SCENARIO=ember-lts-3.4
+    - env: EMBER_TRY_SCENARIO=ember-lts-3.12
+    - env: EMBER_TRY_SCENARIO=ember-lts-3.16
     - env: EMBER_TRY_SCENARIO=ember-release
     - env: EMBER_TRY_SCENARIO=ember-beta
     - env: EMBER_TRY_SCENARIO=ember-canary
     - env: EMBER_TRY_SCENARIO=ember-default-with-jquery
 
-    - stage: "Release"
```
Comment (adrigzr) [inline .travis.yml]: This task must be in in order to be able to release in CI.

**Review by adrigzr — CHANGES_REQUESTED** [review]

**Review by adrigzr — APPROVED** [review]

**Review by adrigzr — CHANGES_REQUESTED** [review]
Set coverage to 100 on `package.json`, it is already done.

File: .eslintrc.js (hunk 1 above)
Comment (svsalvador) [reply .eslintrc.js]: Right

**Review by adrigzr — APPROVED** [review]

**Review by josex2r — APPROVED** [review]

File: tests/dummy/app/controllers/application.js [hunk 4]
```diff
@@ -0,0 +1,16 @@
+import Controller from '@ember/controller';
+import { inject as service } from '@ember/service';
+import { action } from '@ember/object';
+
+export default class ApplciationController extends Controller {
```
Comment (xe21500) [inline tests/dummy/app/controllers/application.js:5]: `Applciation` -> `Application`,  although it will work in both cases

**Review by xe21500 — APPROVED** [review]

**Pull request merged**